Backpack/src/main/res/values/backpack.font.xml

```diff
@@ -1,80 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-
-  <style name="bpkTextBaseRelative" parent="bpkTextBase">
```

Comment:
these are no longer needed - all the new styles didn't have these applied already as its set programmatically with the line height now being included